src/tapis-app/Files/_components/Toolbar/Toolbar.tsx

```diff
@@ -35,14 +35,21 @@ export const ToolbarButton: React.FC<ToolbarButtonProps> = ({
 const Toolbar: React.FC<{ selectedFiles: Array<Files.FileInfo> }> = ({
   selectedFiles,
 }) => {
+  const [isOpen, setIsOpen] = useState(false)
   return (
     <div className={styles.toolbar_wrapper}>
+      <GenericModal
```

Comment:
There should be a separate class that renders this, like `<FilesRenameModal />`. That can be placed at the bottom of the toolbar wrapper.